cmd/manager/kodata/knative-serving-0.6.0.yaml

```diff
@@ -0,0 +1 @@
+../../../config/resources/knative-serving-0.6.0.yaml
```

Comment:
Any problem symlinking to all of `resources/`? There are other manifests in there for setting up a minimal istio controlplane and automatic ingress routes on openshift.

Reply:
Better have the file on a one-on-one basic. We need file under other folder as well, like config/crds/serving_v1alpha1_knativeserving_cr.yaml

Reply:
Yes IMO id be explicit, alternatively I could make a second subdir within kodir and link that to config but as is Id expect this to work just fine.